vale/styles/jargonLint/Technology.yml

```diff
@@ -0,0 +1,325 @@
+---
+action:
+  name: replace
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+swap:
+  # Disable yamlint rule about line length
+  # yamllint disable-line rule:line-length
+  #
+  # Sources
+  # - https://github.com/sapegin/textlint-rule-terminology/blob/master/terms.jsonc
+  # - https://github.com/paradite/frontend-encyclopedia/blob/main/scripts/data.json
+  # - https://github.com/huacnlee/autocorrect/blob/main/.autocorrectrc.template
```

Comment:
Can you separate sources in the Technology.yml file?

As in
```
swap:
  #source1
  a: b
  #source2
  c: d
```

I think that might be better to maintain the list.

Reply:
It won't be easy without a master YAML that will refer the source for each word my problem is the fact that if I split and there is let's say JSON in one, and I find it again in another the rules will have 2 sections.

Some may think the issues will be reported because we will have 2 keys with the sane value, but we may not

# section 1
JSON: JSON

# section 2
(?!\.)JSON: JSON

But we can start building the master file

Reply:
That's by I opened 

- #16

Reply:
Side question, do you think we can merge? 

I mean splitting may require time.

Another option for me, without merging the file is to split the addition to Technology.yaml commit in multiple commits

Reply:
@ccoVeille if we're not going to clarify where each word came from within the rule, then would it be better to remove sources from the rules entirely? Just creating a sources.md file to keep track for each rule, at least temporarily. 

As it is, I don't think it's a good look. We may keep the rules completely clean and add the comments to that separate file.